pkg/apis/batch/validation/validation.go

```diff
@@ -449,7 +449,11 @@ func validateCronJobSpec(spec, oldSpec *batch.CronJobSpec, fldPath *field.Path,
 	if len(spec.Schedule) == 0 {
 		allErrs = append(allErrs, field.Required(fldPath.Child("schedule"), ""))
 	} else {
-		allErrs = append(allErrs, validateScheduleFormat(spec.Schedule, spec.TimeZone, fldPath.Child("schedule"))...)
+		oldSchedule := ""
+		if oldSpec != nil {
+			oldSchedule = oldSpec.Schedule // +k8s:verify-mutation:reason=clone
```

Comment:
drop the `verify-mutation:reason=clone` ... we're not actually cloning anything

rename the variable to something like `existingSchedule` to avoid tripping mutation detection

Reply:
Looks like that one [won't work](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/116252/pull-kubernetes-verify/1699391381060980736), since the way this script is written it matches `= old`, and I don't want to change `oldSpec` in this method, just in case someone will modify it.